src/MongoDB.EntityFrameworkCore/Extensions/MongoDbContextOptionsExtensions.cs

```diff
@@ -28,11 +27,11 @@ namespace Microsoft.EntityFrameworkCore;
 public static class MongoDbContextOptionsExtensions
 {
     /// <summary>
-    /// Configures the <see cref="DbContext"/> subclass to connect to a MongoDB database.
+    /// Configures the context to connect to a MongoDB database.
```

Comment:
Why remove the docs ref here?

Reply:
Was just making them all the consistent text closer to the ones in EF.